.changeset/gold-baboons-sniff.md

```diff
@@ -0,0 +1,5 @@
+---
+"@evmts/bundler": minor
+---
+
+Added persistent cache support. When passed in an optional cache object to the bundler solc results are persisted across compilations
```

Comment:
```suggestion
Added persistent cache support. When passed in an optional cache object to the bundler solc results are persisted across compilations. This is important to add before EVMts vm is added as it will require bytecode. Compiling bytecode can be as much as 80 times slower. This will improve performance in following situations:

1. Importing the same contract in multiple places. With the cache the contract will only be compiled once
2. Compiling in dev mode. Now if contracts don't change previous compilations will be cached
```